packages/transactions/src/ContractUtils.ts

```diff
@@ -52,7 +52,7 @@ class ContractUtils {
   initializeContracts(
     signer: Signer,
     readProvider: ethers.providers.JsonRpcProvider,
-    walletInfo: ISmartAccount,
+    walletInfo: SmartAccountState,
```

Comment:
what was the type earlier?

Reply:
ISmartAccount same as returned by getSmartAccountByOwner. This function called initializeContracts is set up using data we get from getSmartAccountByOwner api consumption.